src/styles/_catalog.scss

```diff
@@ -14,6 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: color 0.3s, transform 0.3s;
+    transition-timing-function: linear;
+
+    &:hover {
+      color: #34568b;
+      transform: scale(1.2);
+      cursor: pointer;
+    }
   }
 
   &__card:not(:nth-child(4n)) {
```

Comment:
Use nesting here more details in  [video](https://mate.academy/learn/css-advanced/sass#/video/925)